content/blog/news/CloudPlugin-GSOC'20.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Google Summer of Code '20 - Cloud Plugin"
+linkTitle: "GSoC'20 : Cloud Plugin"
+date: 2020-08-27
+weight: 1
+description: >
+  A starter to the CueCommander cloud plugin; work done for Google Summer of Code '20
+---
+
+A blog post + starter material for the Cloud Plugin - work done during Google Summer of Code '20 on creating a CueCommander
+ plugin that can be used to manage cloud pools from OpenCue
+ 
+## Summer of Code Proposal: 
```

Comment:
Avoid trailing colons in headers, they aren't really necessary and look weird in the auto-generated TOC on the right side of the page.